examples/ai-image-generator/pages/index.jsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState, useEffect } from "react";
+import initMiro from "../initMiro";
+import PromptInput from "../components/PromptInput";
+import Button from "../components/Button";
+
+export const getServerSideProps = async function getServerSideProps({ req }) {
+  const { miro } = initMiro(req);
+
+  // redirect to auth url if user has not authorized the app
+  if (!(await miro.isAuthorized(""))) {
+    return {
+      props: {
+        boards: [],
+        authUrl: miro.getAuthUrl(),
+      },
+    };
+  }
+
+  const api = miro.as("");
+
+  const boards = [];
+
+  for await (const board of api.getAllBoards()) {
+    boards.push(board.name || "");
+  }
```

Comment:
As I understand, there is no need to load boards

Reply:
yes good point - don't need this.